src/main/java/com/openlattice/edm/set/EntitySetFlag.java

```diff
@@ -5,5 +5,6 @@
     AUDIT,
     EXTERNAL,
     LINKING,
+    MATERIALIZED,
```

Comment:
Are we sticking with "materialized"? I'd be in favor of renaming this to ASSEMBLED or PROJECTED if not